Source/Classes/PINURLSessionManager.m

```diff
@@ -78,6 +78,7 @@ - (void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)task
         dispatch_queue_t delegateQueue = self.delegateQueues[@(task.taskIdentifier)];
     [self unlock];
     
+    if (delegateQueue == nil) return;
```

Comment:
Have you seen this one in the wild? We've only seen it on `didCompleteWithError`: and `didReceiveData:`

Would you mind removing this check?